### PR TITLE
Drops the activity_hints table, if it exists.

### PR DESCRIPTION
**Pull request by ashercodeorg**
This follows on #10331.

Note that this table has approximately 7M rows.

### CODE REVIEW

File: dashboard/db/migrate/20160901180359_drop_activity_hint.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class DropActivityHint < ActiveRecord::Migration[5.0]
```
Comment (Hamms) [inline dashboard/db/migrate/20160901180359_drop_activity_hint.rb:1]: is this the new rails 5.0 syntax?

Comment (ashercodeorg) [reply dashboard/db/migrate/20160901180359_drop_activity_hint.rb:1]: It is. Without that syntax, the following warning occurs.

```
DEPRECATION WARNING: Directly inheriting from ActiveRecord::Migration is deprecated.
Please specify the Rails release the migration was written for:
```

Comment (Hamms) [reply dashboard/db/migrate/20160901180359_drop_activity_hint.rb:1]: hmm. Does this have implications for our old migrations?

Comment (ashercodeorg) [reply dashboard/db/migrate/20160901180359_drop_activity_hint.rb:1]: No, I do not believe so. To start, old migrations should never need to run. And, even should we run them, it is only deprecated (not yet unsupported).

**Comment by Hamms** [comment]
LGTM

**Pull request merged**